packages/sdk-codegen/src/sdkModels.spec.ts

```diff
@@ -172,36 +286,144 @@ describe('sdkModels', () => {
+    }
 
-  describe('sorting it out', () => {
-    it('sorts methods', () => {
-      const list = apiTestModel.methods
-      const keys = Object.keys(list)
-      const sortedKeys = Object.keys(list).sort(localeSort)
-      expect(keys).toEqual(sortedKeys)
-    })
+    it('registers enum types', () => {
+      const types = apiTestModel.types
+      expect(types[titleCase('supported_action_types')]).toBeDefined()
+      expect(types[titleCase('supported_formattings')]).toBeDefined()
+      expect(types[titleCase('pull_request_mode')]).toBeDefined()
+    })
+
+    describe('enum naming', () => {
+      const rf1: OAS.SchemaObject = {
+        name: 'result_format',
+        type: 'string',
+        'x-looker-values': [
+          'inline_json',
+          'json',
+          'json_detail',
+          'json_fe',
+          'csv',
+          'html',
+          'md',
+          'txt',
+          'xlsx',
+          'gsxml',
+        ],
+        description: 'RF1',
+        nullable: true,
+      }
 
-    // it('sorts types', () => {
-    //   const list = apiTestModel.types
-    //   const keys = Object.keys(list)
-    //   const sortedKeys = Object.keys(list).sort(localeSort)
-    //   expect(keys).toEqual(sortedKeys)
-    // })
+      const rf2: OAS.SchemaObject = {
+        name: 'result_format',
+        type: 'string',
+        'x-looker-values': ['pdf', 'png', 'jpeg'],
+        description: 'RF2',
+        nullable: true,
+      }
+
+      const rf3: OAS.SchemaObject = {
+        type: 'string',
+        'x-looker-values': ['csv', 'html', 'txt'],
+        description: 'RF3',
+        nullable: true,
+      }
 
-    it('sorts tags and methods in tags', () => {
-      const list = apiTestModel.tags
-      const keys = Object.keys(list)
-      const sortedKeys = Object.keys(list).sort(localeSort)
-      expect(keys).toEqual(sortedKeys)
-
-      keys.forEach((key) => {
-        const methods = list[key]
-        const methodKeys = Object.keys(methods)
-        const sortedMethodKeys = Object.keys(methods).sort(localeSort)
-        expect(methodKeys).toEqual(sortedMethodKeys)
+      const rf4: OAS.SchemaObject = {
+        name: 'result_format',
+        type: 'string',
+        'x-looker-values': ['csv', 'html', 'txt', 'xlsx'],
+        description: 'RF4',
+        nullable: true,
+      }
+
+      const rf5: OAS.SchemaObject = {
+        name: 'result_format',
+        type: 'string',
+        enum: [
+          'inline_json',
+          'json',
+          'json_detail',
+          'json_fe',
+          'csv',
+          'html',
+          'md',
+          'txt',
+          'xlsx',
+          'gsxml',
+        ],
+        description: 'RF5',
+        nullable: true,
+      }
+
+      it('enum types are renamed and not overwritten', () => {
+        const api = new ApiModel({} as OAS.OpenAPIObject)
+        const actual1 = api.resolveType(rf1)
+        const name1 = titleCase(rf1.name)
+        expect(actual1.name).toEqual(name1)
```

Comment:
I'm more a fan of just using the expected value (e.g. `"Rf1"` in this case ... I think): 
* I find it easier to read the test
* it's an additional check point in case you update `titleCase` impl and its tests but forget some edge case covered here perhaps

Reply:
Fair enough. This was a TDD-ish test I was using to check the renaming rules more than the casing rules. I can hard code the expectation. Pins the logic better.